.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+education, socio-economic status, nationality, personal appearance, race,
+religion, or sexual identity and orientation.
```

Comment:
Is it necessary to include wokeist propaganda? I'd leave out all politics.

Reply:
I just copied it over from the main repo. And i intend to do the same for all other repo's so there's no need switch between repo's all the time. I guess we could refer to the main repo aswell instead of maintaining multiple code_of_conducts and other github specific files